### PR TITLE
[Publishing] @l2succes => Feature header component

### PR DESCRIPTION
**Pull request by kanaabe**
Working in Reaction is really nice! Love how quickly I can test different layouts with storybook. 

Text Layout:
image

Split Layout:
image

Full Layout:
image

TODO
- [x] Replace data-layout garbage
- [x] Configure webpack to accept cors (for fonts). I originally included the font files in /assets, but since they are purchased/licensed, I didn't feel comfortable adding the files in the open.

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<link rel="stylesheet" type="text/css" href="https://d1s2w0upia4e9w.cloudfront.net/fonts/unica-stylesheet.css">
```
Comment (orta) [inline .storybook/preview-head.html]: 👍

**Comment by eessex** [comment]
Don't forget to export the new components here - https://github.com/artsy/reaction/blob/master/src/components/publishing/index.tsx

File: .storybook/middleware.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+module.exports = function expressMiddleware(router) {
+  router.use(
+    "/fonts",
+    proxy({
+      target: "https://d1s2w0upia4e9w.cloudfront.net/",
+      changeOrigin: true,
+    })
+  )
+}
```
Comment (kanaabe) [inline .storybook/middleware.js]: @l2succes thanks for helping me with this!

File: .vscode/settings.json [hunk 3]
```diff
@@ -19,5 +19,6 @@
         "typescriptreact"
     ],
     "prettier.semi": false,
-    "prettier.printWidth": 120
-}
+    "prettier.printWidth": 120,
+    "typescript.tsdk": "./node_modules/typescript/lib"
```
Comment (kanaabe) [inline .vscode/settings.json]: Not sure why this got added -- happy to remove if it's unnecessary.

File: src/components/publishing/feature_header.tsx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+  position: absolute;
+  background-color: black;
+  opacity: 0.17;
+`
+const Title = styled.div`
+  ${Fonts.unica("s100")}
+  margin-bottom: 75px;
+`
+const Vertical = styled.div`
+  ${Fonts.unica("s19", "medium")}
```
Comment (l2succes) [inline src/components/publishing/feature_header.tsx]: nice!

File: src/components/publishing/feature_header.tsx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+  &[data-layout='text'] {
+    ${Title} {
+      margin-bottom: 150px;
+    }
+    ${HeaderText} {
+      max-width: none;
+    }
+  }
+  &[data-layout='split'] {
+    ${Title} {
```
Comment (l2succes) [inline src/components/publishing/feature_header.tsx]: Nice @kanaabe, didn't know you could do that actually. Is that in the docs somewhere?

**Comment by l2succes** [comment]
Ok looks good! Feel to free merge :)

File: src/components/publishing/feature_header.tsx (hunk 5 above)
Comment (kanaabe) [inline src/components/publishing/feature_header.tsx]: Yep! https://www.styled-components.com/docs/faqs#can-i-refer-to-other-components

Comment (alloy) [inline src/components/publishing/feature_header.tsx]: Noice 👌

**Pull request merged**